Scripts/parameters.py

```diff
@@ -1863,7 +1863,11 @@
         "transit": {
             "constant": (0.844179295926 * (3.15335645065 - 0.158), 
                          0.844179295926 * (3.09257722177 - 0.415)),
-            "generation": {},
+            "generation": {
+                "cbd": 0.3,
+                "helsinki_other": 0.3,
+                "espoo_vant_kau": -0.2,
+            },
```

Comment:
I think that if the bug mentioned above is fixed, the dummy parameters can be easily modified like this:
`"cbd": 0.3+0.3-0.2,
"helsinki_other": 0.3-0.2,
"espoo_vant_kau": -0.2,`
because I think that is the effect right now.

Reply:
True, need to try this first